src/index.md

```diff
@@ -8,6 +8,19 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+save your website's files locally, on your users' devices. Consider Workbox if
+you want to:
+
+* Make your site work offline.
+* Improve load performance. Even if you don't want to go fully-offline, you can
+  use Workbox to save and serve common files locally, rather than from the
+  network.
```

Comment:
Do you think we might want to be clear that we're talking about repeat-view performance or future visit performance rather than first view load perf in general?